client/scripts/views/components/header.ts

```diff
@@ -105,6 +105,13 @@ const Header: m.Component<{}> = {
             iconLeft: Icons.SETTINGS,
             label: 'Settings'
           }),
+          (app.vm.activeAccount
+            || m.route.get() === '/settings')
+            && m(MenuItem, {
+              onclick: () => m.route.set('/notifications'),
+              iconLeft: Icons.BELL,
+              label: 'Manage Notifications'
+            }),
```

Comment:
@gdjohnson @zakhap this will need to be added to the psuedoheader that Graham implemented

Reply:
@zakhap can you pull in the latest changes to raymond.construct and add the manage notifications section to the pseudoheader

Reply:
Yes I can